vtrace/__init__.py

```diff
@@ -708,6 +713,20 @@ def removeBreakpoint(self, id):
         # Remove cached breakpoint code
         Breakpoint.bpcodeobj.pop(id, None)
 
+    def _updateBreakAddresses(self):
+        """
+        Update breakpoint address resolution (in unresolved breakpoints).
+        Intended to be run after events which change the namespace, such as
+        NOTIFY_LOAD_LIBRARY events
+        """
+        for bp in self.deferred:
```

Comment:
i totally get that.  i think it's because we have VDB's tracer objects and the raw traces from vtrace.  we can talk about any rearchitecting you think might be beneficial.

Reply:
```
class Linuxi386Trace(
        vtrace.Trace,
        LinuxMixin,
        v_i386.i386Mixin,
        v_posix.ElfMixin,
        v_base.TracerBase):
...
    def __init__(self):
        vtrace.Trace.__init__(self)
        v_base.TracerBase.__init__(self)
        v_posix.ElfMixin.__init__(self)
        v_i386.i386Mixin.__init__(self)
        LinuxMixin.__init__(self)
```